/**:
  ros__parameters:

    in_speed: 0.50 # Скорость после знака дорожных работ
    out_speed: 0.40 # Скорость после завершения испытания

    angle_L: 50.0  # Угол поворота налево
    angle_R: -70.0 # Угол поворота направо

    linear_x_L: 0.15  # Линейная скорость при повороте налево
    linear_x_R: 0.175 # Линейная скорость при повороте направо

    angular_z_L: 1.0 # Угловая скорость при повороте налево
    angular_z_R: 1.0 # Угловая скорость при повороте направо

    distance: 0.55 # Дистанция для начала поворота